filters/fadein/fadein.go

```diff
@@ -229,13 +228,7 @@ func (p *postProcessor) Do(r []*routing.Route) []*routing.Route {
 		for i := range ri.LBEndpoints {
 			ep := &ri.LBEndpoints[i]
 
-			s, h, err := normalizeSchemeHost(ep.Scheme, ep.Host)
```

Comment:
Should we remove normalizeSchemeHost since it is not used here anymore?

Reply:
No, we should keep it, because it's still used in the `func (endpointCreated( CreateFilter ...`.
This function parses filter args from dataclient, so basically it parses user input which should be normalized as well.